go.mod

```diff
@@ -1,11 +1,12 @@
 module tinygo.org/x/bluetooth
 
-go 1.18
+go 1.20
```

Comment:
Is this intentional? Do we need Go 1.20 for something?

Reply:
This is due to the cyw43439 package requiring that version: https://github.com/soypat/cyw43439/blob/main/go.mod#L3

see https://github.com/soypat/cyw43439/issues/48